### ISSUE TITLE
switch to conda version of minc-toolkit?

### ISSUE THREAD

**Issue by vfonov** [gdevenyi/NeuroAnsible#25]
I have recipes for building minc-toolkit with conda, and there is version 1.9.18 based on the classic anaconda packages : https://anaconda.org/vfonov/minc-toolkit-v2 and also one based on conda-forge in https://anaconda.org/vfonov/repo/files?type=any&label=minc-forge 

I started to create a minimally dependant packages  (mostly for python) to link only to libminc ( https://anaconda.org/vfonov/libminc ) to not drag all the minc-toolkit dependencies.

**Comment by gdevenyi** [comment]
I had on my list to attempt some maintenance/release work on minc-toolkit. I would be ok with moving to a conda-based minc, but what are the chances to upstreaming to conda-forge?

**Comment by vfonov** [comment]
well... I think minc-toolkit is too big of a monster for conda-forge , if we break it up into subpackets, it might work

**Comment by gdevenyi** [comment]
Yes of course, I mean the subpackages, starting with libminc would be the big one.

**Comment by vfonov** [comment]
https://github.com/vfonov/conda-recipes-forge - here are my recipes. I could try to upstream a couple of simple ones.
